sim/compare_snps.py

```diff
@@ -75,20 +103,19 @@ def analyse(simulated_snps, pipeline_snps, mask_filepath):
         "TP": tp,
         "FP": fp,
         "FN": fn,
-        "masked TPs": tp_in_mask,
-        "masked FPs": fp_in_mask, 
-        "masked FNs": fn_in_mask, 
+        "misssing sites": n,
+        "TPs in mask": tp_in_mask,
+        "FPs in mask": fp_in_mask, 
+        "FNs in mask": fn_in_mask, 
```

Comment:
Please remove spaces in for the dictionary key names. Spaces are generally more awkward to deal with in this context